be/src/block_cache/star_cachelib.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "block_cache/kv_cache.h"
+#include "common/status.h"
+#include "submodules/starcache/src/star_cache.h"
+
+namespace starrocks {
+
+class StarCacheLib : public KvCache {
```

Comment:
Is `StarCacheWrapper` more appropriate?

Reply:
> Is `StarCacheWrapper` more appropriate?

As we use the `FbCacheLib` in the pervious version, so, I will change these two class names to `CacheLibWrapper` and `StarCacheWrapper`.